meson.build

```diff
@@ -324,6 +330,24 @@ if gnutls.found()
   conf.set('HAVE_GNUTLS', '1')
 endif
 
+libgbinder = dependency('libgbinder', version: '>= 1.1.0', required: get_option('binder'))
```

Comment:
are we always going to be doing a cross build to get android?  Or will there ever be native builds?